lib/absinthe/federation/schema/entities_field.ex

```diff
@@ -171,6 +176,23 @@ defmodule Absinthe.Federation.Schema.EntitiesField do
     end
   end
 
+  defp convert_key_to_atom(map) when is_map(map) do
+    map
+    |> Enum.reduce(%{}, fn {k, v}, acc ->
+      k = k |> convert_key()
```

Comment:
prefer passing the arg without a pipe when there is only one pipe
```suggestion
      k = convert_key(k)
```

Reply:
I found you are flowing the credo rules, Do we need to add the `credo` deps in [:dev, : test]?